src/components/Register/RegisterForm.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { registerUser, setTokens } from 'api/auth.service';
-import history from 'containers/history'
+import { Redirect } from 'react-router';
```

Comment:
Probably don't need this import, I think it was from a previous commit I made while trying to solution that history issue